src/CarbonAwareSDK.sln

```diff
@@ -53,6 +53,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CarbonAware.DataSources.Jso
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CarbonAware.DataSources.WattTime.Mocks", "CarbonAware.DataSources\CarbonAware.DataSources.WattTime\mock\CarbonAware.DataSources.WattTime.Mocks.csproj", "{69DA1C7F-740E-406D-B62F-7E84175378B2}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GSF.CarbonAware", "GSF.CarbonIntensity\src\GSF.CarbonAware.csproj", "{66BFF3BC-F15B-4A56-B8E4-6CE68EAF54CD}"
```

Comment:
Is it missing to rename `GSF.CarbonIntensity` dir to the new name?

Reply:
FIxed